comfy_cli/locking.py

```
@@ -0,0 +1,147 @@
+            _release(fd)
+    finally:
+        os.close(fd)
+
+
+if sys.platform == "win32":
+    import msvcrt  # type: ignore[import-not-found]
+
+    def _acquire(fd: int, *, timeout: float | None) -> None:  # pragma: no cover - platform branch
+        # Lock a single byte at offset 0. msvcrt has no native timeout, so we
+        # spin with short non-blocking attempts.
+        import time
```

Pylint (4.0.5)

[convention] Import outside toplevel (time)

(C0415)